### ISSUE TITLE
Announcement Bar: Contrast

### ISSUE THREAD

**Issue by syarwick23** [loyolauniversitychicago/loyola-template-4.0#102]
When text is bold or italicized, the text turns black on the maroon background, which does not meet a 4.5:1 contrast ratio. 
ADDITIONALLY, linked text turns maroon and is essentially invisible to users.

I would recommend make sure bold and italicized text is white, and links are gold. 	

Since this content piece uses a HTML textbox, users can enter anything. I'd recommend just using a Plain Text box if we want to avoid problems.
